docs/gno-tooling/cli/gnokey/full-security-tx.md

```diff
@@ -0,0 +1,128 @@
+---
+id: full-security-tx
+---
+
+# Making an airgapped transaction
+
+## Prerequisites
+
+- **`gnokey` installed.** Reference the
+  [Local Setup](../../../getting-started/local-setup/installation.md#2-installing-the-required-tools) guide for steps
+
+## Overview
+
+`gnokey` provides a way to create a transaction, sign it, and later
+broadcast it to a chain in the most secure fashion. This approach, while more 
+complicated, grants full control and provides airgap support.
+
+The indented purpose of this functionality is to provide maximum security when 
+signing and broadcasting a transaction. In practice, this procedure should take
+place on two separate machines, one with access to the internet (`Machine 1`), 
+and the other one without (`Machine 2`), with the separation of steps as follows:
+1. `Machine 1`: Fetch account information from the chain
+2. `Machine 2`: Create an unsigned transaction locally
+3. `Machine 2`: Sign the transaction
+4. `Machine 1`: Broadcast the transaction
```

Comment:
A and B are good, but it does not make sense to have alice & bob, since these are two actors. Airgap still happens under the control of one single actor, just with their two machines. Renamed it to A and B: [7a2010d](https://github.com/gnolang/gno/pull/2226/commits/7a2010d7c349bf041c9447cc3eee30cb4eded455)

Reply:
Alright, cool! 👍